forms/CommentCreateForm.php

```diff
@@ -67,9 +70,11 @@ public function save()
         $Comment = $this->Comment;
 
         if (empty($this->id)) {
-            $Comment = new Comments\models\Comment();
+            $Comment =  \Yii::$app->getModule(Comments\Module::NAME)->model('comment');
```

Comment:
I think that I haven't made myself clear in what I was trying to say in the last comment.

What I'm trying to say it's that I'll add the createObject support in each call instead of the ones there are right now, for example the code you have psoted would be:

```php
         if (empty($this->id)) {
-            $Comment = new Comments\models\Comment();
+            $Comment =  \Yii::createObject(Comments\models\Comment::className());
```

So I'll be using DI in everything right now instead the code I have sent to you and change the Module to its former state.

But I also think that it would be an interesting "syntactic sugar" to add the module support for models, since you are already initializing the module each time there is a call to the widgets (or, at least, if there are comments in the panel), so nothing wrong there; bootstraping wouldn't the best decision in this case because probably users wont have comments in every page, I'm not a friend of having modules loading each time I open an application page.

But as I have said, that's only syntactic sugar, no problem there. Tell you what:

* I'll add only the createObject in each calling,
* If you are OK with that, I'll add the model support to the module, if not, I won't, it's only syntactic sugar to simplify module management, no problem there.